plugin.json

```diff
@@ -57,6 +57,12 @@
         "display_name": "Application ID",
         "type": "text",
         "help_text": "Application ID from the Azure Active Directory. Only required for Skype for Business Online. Use this URL to configure the plugin in Azure Active Directory: ```https://SITEURL/plugins/skype4business/api/v1/auth_redirect```"
+      },
+      {
+        "key": "LogClientRequests",
+        "display_name": "Enable logging requests to S4B server",
+        "type": "bool",
+        "help_text": "Logging requests sent to S4B server for debugging purposes. Used only with Skype for Business Server."
```

Comment:
Maybe put a note in here that these logs will use the `INFO` log level, so the server's log level must be set properly for this to work.

Reply:
It definitely makes sense. I've just added a note :+1: